devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -30,35 +46,71 @@ pipelineWrapper({
+        String u_id = utils.shStdout('id -u')
+        docker.build(
+                "$projectName:debs-upload",
+                "--build-arg=u_id=$u_id -f devops/docker/cd/debs_upload.dockerfile devops/docker/cd"
+        ).inside {
+            withCredentials([file(credentialsId: 'SovrinRepoSSHKey', variable: 'sovrin_key')]) {
+                sh """
+                    ./$pkgUtilsDir/upload_debs.py $debsDir $repoName $repoComponent --host $env.SOVRIN_REPO_HOST --ssh-key $sovrin_key
+                """
+            }
+        }
+    }
+
+    Closure _notify = {
+        notifier.email {
+            to = '$DEFAULT_RECIPIENTS ' + (this.env.LIBSOVTOKEN_DEF_RECIPIENTS ?: '')
+            subject = "[$projectName] new packages ${pkgsList.join(' ')} published to '${repoName}/${repoComponent}' repo"
+            body = """
+                |New debian packages ${pkgsList.join(' ')} were built and published to '${repoName}/${repoComponent}' repo
+                |
+                |Build:
+                |   Project: \$PROJECT_NAME
+                |   Url: ${this.env.BUILD_URL}
+                |   Number: ${this.env.BUILD_NUMBER}
+                |
+                |Check console output at ${this.env.BUILD_URL}console to view the details.
+            """.stripMargin()
+        }
+    }
+
     nodeWrapper(nodeLabels.codeBuild) {
         def osname = 'xenial'
         List projects = ['sovtoken', 'sovtokenfees']
-        List pkgsList = []
         String indyNodeVersion
-        Boolean isRC = false
-        Boolean isRelease = false
-        Boolean isDev = (env.BRANCH_NAME == 'master')
-
-        stage('Checkout sources from SCM') {
-            checkout scm
-        }
 
         def sovrinRepo = sovLibrary.SovrinRepo.new(this)
-        def utils = sovLibrary.Utils.new(this)
         def buildCtx = sovLibrary.AwsCodeBuildHelper.BuildCtx.new('plugin',
             ['devops', 'sovtoken/sovtoken/__metadata__.py', 'sovtokenfees/sovtokenfees/__metadata__.py'])
         def awsCBHelper = sovLibrary.AwsCodeBuildHelper.new(this, buildCtx)
 
-        stage('Get indy-node version') {
+        stage('Checkout sources from SCM') {
+            checkout scm
+        }
+
+        stage('Set release params') {
             indyNodeVersion = utils.shStdout("""
                 sed -n "s/.*indy-node==\\([^\\"\\']\\+\\).*/\\1/p" sovtoken/setup.py
             """)
             if (!indyNodeVersion) {
                 throw new Exception('indy-node version is not found')
             }
-            isRC = !!((env.BRANCH_NAME == 'stable') && indyNodeVersion.find(/[^0-9\.]/))
-            isRelease = ((env.BRANCH_NAME == 'stable') && !isRC)
-            logger.info("Detected indy-node version: '$indyNodeVersion', isRC: $isRC, isRelease: $isRelease")
+            logger.info("Detected indy-node version: '$indyNodeVersion'")
+
+            isRC = (env.BRANCH_NAME == 'stable')
+            isRelease = !!(isRC && !indyNodeVersion.find(/[^0-9\.]/))
```

Comment:
It would be great to have a comment on true meaning of those flags or make their names more precise (i.e. `isRC` doesn't necessarily mean that we're building RC, which is kind of counter-intuitive)